backend/follows/api/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
-from .views import createFollowAPI, viewFollowersAPI, viewFollowingsAPI
+from .views import createFollowAPI, viewFollowersAPI, viewFollowingsAPI, deleteFollowAPI
 
 urlpatterns = [
     path('createFollow', createFollowAPI.as_view(), name='createFollow'),
-    path('followers', viewFollowersAPI.as_view(), name='followers'),
-    path('followings', viewFollowingsAPI.as_view(), name='followings'),
+    path('followers/<int:pk>', viewFollowersAPI.as_view(), name='followers'),
+    path('followings/<int:pk>', viewFollowingsAPI.as_view(), name='followings'),
+    path('deleteFollow', deleteFollowAPI.as_view(), name='deleteFollow'),
```

Comment:
1. No need to mention follow. It is understandable like this if the api url is /follows/create or /follows/delete
2. Also no need for name coz i think it is only needed in django environment for redirection
3. You can call the url directly with pk and no need to pass who is following or follower. It will improve the performance.

```suggestion
    path('create', createFollowAPI.as_view()),
    path('<pk>/delete', deleteFollowAPI.as_view()),
```